src/compare/index.ts

```diff
@@ -1,18 +1,18 @@
 import * as github from "@actions/github";
-import {Benchmark, StorageOptions} from "../types.js";
-import {getGithubEventData, GithubActionsEventData, parseBranchFromRef, getDefaultBranch} from "../utils/index.js";
 import {isGaRun} from "../github/context.js";
 import {IHistoryProvider} from "../history/provider.js";
 import {validateBenchmark} from "../history/schema.js";
+import {Benchmark, StorageOptions} from "../types.js";
+import {GithubActionsEventData, getDefaultBranch, getGithubEventData, parseBranchFromRef} from "../utils/index.js";
 
-const CompareWithTypeEnum = {
+const compareWithTypeEnum = {
```

Comment:
I forget already... Aren't these suppose to be PascalCase?

```suggestion
const CompareWithTypeEnum = {
```